### PR TITLE
fixes #10986 - ensure noop is defined in upgrade hook

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: hooks/post/30-upgrade.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 if app_value(:upgrade)
+  noop = app_value(:noop) ? ' (noop)' : ''
```
Comment (ehelms) [inline hooks/post/30-upgrade.rb]: How does moving it earlier help ensure its defined?

Comment (stbenjam) [inline hooks/post/30-upgrade.rb]: https://github.com/Katello/katello-installer/pull/236/files#diff-1453f17351c9fefb73f41a2dd0837b82R5 references it, and it was before it was defined.  Do you want a better commit message maybe?

Comment (ehelms) [inline hooks/post/30-upgrade.rb]: Ahh, yea thanks for pointing that out. I figured there was something I just wasn't seeing it.

**Comment by ehelms** [comment]
APJ

**Comment by stbenjam** [comment]
[test]

**Comment by ehelms** [comment]
Since this is only used for upgrade, and we don't test upgrades through the CI system I am ok with this being merged.

**Comment by stbenjam** [comment]
Ok, thanks @ehelms

**Pull request merged**